src/lab15.py

```diff
@@ -283,7 +283,7 @@ def add_inline_child(self, node, w, child_class, frame, word=None):
         line = self.children[-1]
         previous_word = line.children[-1] if line.children else None
         if word:
-            child = child_class(node, line, previous_word, word)
+            child = child_class(node, word, line, previous_word)
```

Comment:
Was this a real bug in the code?

Reply:
Ah, no, ok, here's what's going on. Between Ch 14 and 15, we change where the `word` argument goes in the argument order. In 14 and prior, the `word` argument goes after `node`; in 15 and 16, it goes at the end. This came about due to various refactors, but isn't necessary now, so I'm fixing it so the argument order doesn't change for no reason.